config/_electrum_urls/mainnet

```diff
@@ -1 +1,4 @@
-ssl://electrum.blockstream.info:50002
\ No newline at end of file
+wss://electrumx-server.tbtc.network:8443
+wss://electrum.boar.network:2083
+wss://bitcoin.threshold.p2p.org:50004
+ssl://electrum.blockstream.info:50002
```

Comment:
Should we include all those servers in tests executed in `/pkg/bitcoin/electrum/electrum_integration_test.go`? 

Checking if the connection works is fine but checking if they can execute e.g. `GetLatestBlockHeight` is even better.

Reply:
Yes, we should have a way to validate the servers, it's tracked in https://github.com/keep-network/keep-core/issues/3594.